### PR TITLE
Add Config:recent_posts_limits

### PR DESCRIPTION
**Pull request by RammusXu**
Hi, I think recent_posts_limits need more flexible. So I add this. Please check it.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ banner: "images/banner.jpg"
 # Content
 excerpt_link: Read More
 fancybox: true
+recent_posts_limits: 10
```
[the remark this answers is not in the document]
Comment (RammusXu) [reply _config.yml]: It just too less for me😞
10 is more pretty number 🤣

[the remark this answers is not in the document]
Comment (YoshinoriN) [reply _config.yml]: Thanks :) 
I think maybe current `5` is also nothing meanings, but would you please fix it to `5` if you have nothing other reasons.

File: README.md [hunk 2]
```diff
@@ -59,6 +60,7 @@ twitter:
 - **banner** - Path of title banner image of page top
 - **excerpt_link** - "Read More" link at the bottom of excerpted articles. `false` to hide the link.
 - **fancybox** - Enable [Fancybox]
+- **recent_posts_limits** - How many post in Home page.
```
Comment (YoshinoriN) [inline README.md]: Would you please write more detail?

**Comment by YoshinoriN** [comment]
Thanks a lot :)

**Pull request merged**